pepper-apis/src/main/java/org/broadinstitute/ddp/route/PostPasswordResetRoute.java

```diff
@@ -62,15 +66,15 @@ public Object handle(Request request, Response response) throws Exception {
             }
         }
 
-        HttpUrl.Builder urlBuilder = clientPwdResetUrl.newBuilder();
+        var urlBuilder = new URIBuilder(clientPwdResetUrl);
```

Comment:
Since the new `HttpUrl` does proper encoding now, I took the opportunity to replace it with `URIBuilder` which also does proper encoding. The advantage here is that `URIBuilder` comes from the Apache `httpclient` library which we have a direct dependency for versus `HttpUrl` which is a transitive dependency.

Reply:
Nice!